recipes/pycoalescence/bld.bat

```diff
@@ -0,0 +1,13 @@
+cd pycoalescence
+
+%PYTHON% installer.py --cmake-args="-DCMAKE_PREFIX_PATH=%PREFIX% -DCMAKE_INSTALL_PREFIX=%PREFIX% -DCMAKE_C_COMPILER=%CC% -DCMAKE_CXX_COMPILER=%CXX%"
+
+del __pycache__
+del obj
+del reference
+del lib
+
+cd ..
+
+copy pycoalescence %SP_DIR%
```

Comment:
This is non-standard and hard to maintain.
I'm OK merging this but it would be nice to upstream a better build script for this package.

Reply:
Definitely something I'm planning on. I think it should all be possible just through meta.yaml in the future.